src/sentry/tasks/integrations.py

```diff
@@ -154,3 +156,66 @@ def kick_off_status_syncs(project_id, group_id, **kwargs):
                 'external_issue_id': external_issue_id,
             }
         )
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.kickoff_vsts_subscription_check',
+    queue='integrations',
+    default_retry_delay=60 * 5,  # TODO(lb): not sure what this should be
+    max_retries=5,
+)
+@retry()
+def kickoff_vsts_subscription_check():
+    organization_integrations = OrganizationIntegration.objects.filter(
+        integration__provider='vsts',
+        # integration__status=ObjectStatus.VISIBLE,
+        # status=ObjectStatus.VISIBLE,
+    ).select_related('integration')
+    update_interval = time() - 60 * 60 * 60 * 6
```

Comment:
Maybe we do something like

```python
datetime.now() - timedelta(hours=6)
```

just to make this a little more readable than the time math? Would probably need to change line 220 to `datetime.now()`.

Could also name it something like `six_hours_ago` so that line 184 reads:

```python
subscription['check'] > six_hours_ago
```

Not _necessary_ changes, just readability suggestions.